kahuna/public/js/image/service.js

```diff
@@ -38,7 +38,8 @@ imageService.factory('imageService', ['editsService', function(editsService) {
         return {
             cost: image.data.cost,
             hasCrops: hasExportsOfType(image, 'crop'),
-            isValid: image.data.valid
+            isValid: image.data.valid,
+            canDelete: image.getAction('delete').then(action => !! action)
         };
```

Comment:
I think we should rework this service soon, it doesn't make sense to do all the work it does if you're only interested in a single one of them.